spec/unit/webrtc/groupCall.spec.ts

```diff
@@ -698,7 +662,7 @@ describe('Group Call', function() {
 
                 expect(client1.sendToDevice).toHaveBeenCalled();
 
-                const oldCall = groupCall1.calls.get(
+                const oldCall = (groupCall1 as any).calls.get(
```

Comment:
When accessing private fields we should always prefer a `ts-ignore` comment over casting to `any`, since `ts-ignore` preserves type information